assets/src/web-stories-block/block/components/storyCard.js

```diff
@@ -48,19 +48,17 @@ function StoryCard({
   const dateFormat = __experimentalGetSettings().formats.date;
 
   return (
-    <div className={singleStoryClasses}>
+    <div className={singleStoryClasses} ref={itemRef}>
       <div className="web-stories-list__story-poster">
         <img src={poster} alt={title} />
       </div>
       {hasContentOverlay && (
         <div className="story-content-overlay web-stories-list__story-content-overlay">
           {isShowingTitle && title && (
-            <RawHTML className="story-content-overlay__title">{title}</RawHTML>
+            <div className="story-content-overlay__title">{title}</div>
           )}
           {isShowingExcerpt && excerpt && (
-            <RawHTML className="story-content-overlay__excerpt">
-              {excerpt}
-            </RawHTML>
+            <div className="story-content-overlay__excerpt">{excerpt}</div>
```

Comment:
What's the reason for not using `RawHTML` anymore?

Reply:
I had introduced this change as a test, and since reverted as well, guess my push didn't go through. We do need it as `excerpt` contains an html string.